### PR TITLE
OCPBUGS#22777: Updated GCP Kubernetes manifest optional clause

### PR DESCRIPTION
**Pull request by dfitzmau**
[OCPBUGS-22777](https://issues.redhat.com/browse/OCPBUGS-22777)

Version(s):
4.15 and 4.14

Link to docs preview:
[Creating the Kubernetes manifest and Ignition config files](https://68975--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installing-gcp-user-infra#installation-user-infra-generate-k8s-manifest-ignition_installing-gcp-user-infra)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/installation-user-infra-generate-k8s-manifest-ignition.adoc [hunk 1]
```diff
@@ -193,6 +193,13 @@ ifdef::aws,azure,ash,gcp[]
 ----
 $ rm -f <installation_directory>/openshift/99_openshift-cluster-api_worker-machineset-*.yaml
 ----
+ifndef::user-infra-vpc[]
++
+[IMPORTANT]
+====
+If you did not enable the Machine API for your cluster, you must remove the Kubernetes manifest files that define the worker machines. Otherwise, your cluster fails to install.
```
Comment (jinyunma) [inline modules/installation-user-infra-generate-k8s-manifest-ignition.adoc:200]: suggest to update `Machine API` to `capability MachineAPI`

Comment (dfitzmau) [reply modules/installation-user-infra-generate-k8s-manifest-ignition.adoc]: Thanks, Jinyun. I see that capability exists in our [docs](https://docs.openshift.com/container-platform/4.14/installing/cluster-capabilities.html#machine-api-capability_cluster-capabilities). Thanks for pointing this out. I updated the clause accordingly. Please let me know your thoughts.

Comment (jinyunma) [reply modules/installation-user-infra-generate-k8s-manifest-ignition.adoc]: LGTM

**Comment by jianli-wei** [comment]
@dfitzmau According to Jinyun's [comment](https://issues.redhat.com/browse/OCPBUGS-22777?focusedId=23600363&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-23600363), shall the versions be 4.14 - 4.15 instead? Thanks!

**Comment by dfitzmau** [comment]
> @dfitzmau According to Jinyun's [comment](https://issues.redhat.com/browse/OCPBUGS-22777?focusedId=23600363&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-23600363), shall the versions be 4.14 - 4.15 instead? Thanks!

Thanks, @jianli-wei . Well spotted. The PR now targets 4.14 and 4.15.

**Comment by kcarmichael08** [comment]
LGTM!

**Pull request merged**